modules/ROOT/pages/creating-an-odatav4-api-with-apikit.adoc

```diff
@@ -101,14 +101,156 @@ image::add-db-transform-to-sql.png["Select element and tab are highlighted, as w
 
 image::serialize-transform-to-sql.png["Serialized entity element and tab are highlighted, as well as the inbound content in the General section."]
 
+=== What is transform to SQL select in OData 4?
+
+The `transform-to-sql-select` is an operation that allows the user to transform an OData query request into a SQL select statement. It has been designed to work with the Mule DB connector to help the user expose databases with OData 4 easily.
```

Comment:
This says pretty much the same as the introduction that already exists, is it worth to merge both or keep only one in some way? or do you think that is worth to refresh what the operation is meant to do in this new section?

Reply:
You may be right that both paragraphs may express the same idea with different words.
But the first paragraph is meant to describe the transform to SQL operation, and the second paragraph is meant to engage the reader to use such an operation by explaining that it works with the Mule DB connector and providing the elements it supports.
Nevertheless, we may merge both things and change the article layout to:

Use The Transform to SQL Operation
APIkit for OData 4 contains a Transform to SQL Select operation to transform the input OData GET request to a SQL SELECT query. The operation is useful when combined with the select operation of the Database Connector. The output payload is a prepared statement and the attributes are a map of prepared statement parameters. It has been designed to work with the Mule DB connector to help the user expose databases with OData 4 easily.
Image 1 
Image 2
Image 3
On the one hand, APIkit currently supports the following elements:
Etc.

Do you think this change would improve this article?